pyvista/utilities/reader.py

```diff
@@ -1004,6 +1004,53 @@ def all_patch_arrays_status(self):
         return {name: self.patch_array_status(name) for name in self.patch_array_names}
 
 
+class POpenFOAMReader(OpenFOAMReader):
+    """Parallel OpenFOAM Reader for .foam files.
+
+    Can read parallel-decomposed mesh information and time dependent data.
```

Comment:
I think we should document in the description that this reader can also used for parallel reconstructed data and non-parallel data.